installer/templates/phx_umbrella/apps/app_name_web/README.md

```diff
@@ -3,7 +3,7 @@
 To start your Phoenix app:
 
   * Install dependencies with `mix deps.get`
-  * Create and migrate your database with `mix ecto.create && mix ecto.migrate`
+  * Setup your database with `mix ecto.setup`
```

Comment:
Yeah, I should have written something. Honestly I don't know what's the best answer here, I'm sure maintainers will chime in. I like the `ecto.setup` change in non-umbrella project though!